crates/electrum/Cargo.toml

```diff
@@ -12,6 +12,11 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bdk_chain = { path = "../chain", version = "0.6.0", default-features = false }
+bdk_chain = { path = "../chain", version = "0.6.0", features = [ "miniscript" ] }
```

Comment:
This will force the crate to use `bdk_chain` with default features and `"miniscript"` when it's not strictly needed.

Can you revert this and add a `bdk_chain` dependency under `[dev-dependencies]`?

Reply:
Why do we need miniscript to test this crate?

Reply:
Including `miniscript` was to get a very early version of `test_electrum` to work properly. It is no longer needed, but I forgot to remove it.